code/controllers/configuration/entries/general.dm

```diff
@@ -291,7 +291,7 @@
 /datum/config_entry/string/banappeals
 
 /datum/config_entry/string/wikiurl
-	default = "http://www.tgstation13.org/wiki"
+	default = "https://wiki.skyrat13.tk/w/index.php" //SKYRAT EDIT - Original: "http://www.tgstation13.org/wiki"
```

Comment:
Something like that might actually benefit from being changed in `config.txt`, no? Unless you're telling me you tested this in-game and it worked, in which case it's fine.

Reply:
Oof I thought config.txt was local, didn't think the change would go forward to the server correctly if I did it from github 😅 
I can get a commit up in a second